### ISSUE TITLE
[Select one/many - Other] Focus on "other" text field when other option is selected

### ISSUE THREAD

**Issue by gino-m** [google/ground-android#2403]
@justinkimtang Could you PTAL? @jcqli FYI

**Comment by angrezichatterbox** [comment]
Is this issue open to be fixed by other .. Could I fix this issue

**Comment by gino-m** [comment]
@angrezichatterbox Sure, please go ahead!

**Comment by angrezichatterbox** [comment]
The app seems to crash when I click on sign-in with google . Is it a problem from my side ?

**Comment by gino-m** [comment]
> The app seems to crash when I click on sign-in with google . Is it a problem from my side ?

Sorry for the delay. Are you running the app against local Firebase emulator?

**Comment by angrezichatterbox** [comment]
I am running on an android simulator installed on my laptop

**Comment by gino-m** [comment]
@angrezichatterbox Please be sure you're building `localDebug` variant and following https://github.com/google/ground-platform/wiki/Ground-Developer's-Guide to get started. Let me know if that helps!

**Comment by angrezichatterbox** [comment]
I will try running it and solving the issue by following the guide properly. Thanks :)

**Comment by anandwana001** [comment]
Hi @angrezichatterbox 
Are you still facing the issue?

Happy to help you set up the app.

**Comment by angrezichatterbox** [comment]
Yes I am not able to launch the firebase emulator and set it up . I tried launching but I am facing some errors.

**Comment by anandwana001** [comment]
Quick Questions @angrezichatterbox 
1. Have you been able to put the google-services.json correctly?
2. Were you able to run a local server?

**Comment by angrezichatterbox** [comment]
I will try setting it up once again properly. I haven't yet worked with firebase emulator.

**Comment by anandwana001** [comment]
Feel free to let me know if you need to get on a call and wrap up this initial process.

**Comment by jcqli** [comment]
@angrezichatterbox Any updates?

**Comment by angrezichatterbox** [comment]
I will be trying to setup the firebase once again with the help of my friend and start working on this. if I am still unable to work on it i will unassign myself.

**Comment by angrezichatterbox** [comment]
I am now able to run the app. Could I get note of what to solve exactly since the screen that's currently visible to me is no  surveys available.

**Comment by gino-m** [comment]
> I am now able to run the app. Could I get note of what to solve exactly since the screen that's currently visible to me is no surveys available.
> 
> <img alt="Screenshot 2024-08-23 at 7 04 28 PM" width="265" src="https://private-user-images.githubusercontent.com/62511283/360944991-dacab74c-4812-4ba3-be03-1f75da2459dc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ0MjE2MDgsIm5iZiI6MTcyNDQyMTMwOCwicGF0aCI6Ii82MjUxMTI4My8zNjA5NDQ5OTEtZGFjYWI3NGMtNDgxMi00YmEzLWJlMDMtMWY3NWRhMjQ1OWRjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIzVDEzNTUwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRkNWQ3NmU4YmE4ZjBhNDU4NjhiMWNhYzYwNGUzNjIyODkyMTU4OTQ4ODEyYzRhOGI3NGNlMzVmNzhkMzI5NTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.y56RHBddZxAx2bly_nj5DCtg5uzl68BnRtJWX2V-A3Y">

You need to first create a survey via the web app. When you reach the screen when you define tasks, add a "Multiple choice" task with an "Other" option. See https://github.com/google/ground-platform/wiki/Ground-Developer%27s-Guide for detailed instructions on running the web app locally.

**Comment by angrezichatterbox** [comment]
I have ran the web locally and made a survey. Is it normal to see an empty map when opening the survey. How exactly to see the survey question and option. My apologies if I am asking too much questions.

**Comment by jcqli** [comment]
@angrezichatterbox Are you still facing issues setting up your environment?

**Comment by angrezichatterbox** [comment]
> @angrezichatterbox Are you still facing issues setting up your environment?

I set up the environment properly. I logged into the app and then was doing the feedback I was not able to select any place so that I could continue in the app since the map was empty.

**Comment by gino-m** [comment]
> I have ran the web locally and made a survey. Is it normal to see an empty map when opening the survey. How exactly to see the survey question and option. My apologies if I am asking too much questions.

Sorry I missed this. Not sure you can use the map when running locally. If you set up against your own Firebase instance it should work. 

That said, you should be able to add a point or polygon - just make sure you've selected "Free form" data collection for at least one job when setting up your survey on the web interface.

**Comment by angrezichatterbox** [comment]
> > I have run the web locally and made a survey. Is it normal to see an empty map when opening the survey. How exactly to see the survey question and option. My apologies if I am asking too much questions.
> 
> Sorry I missed this. Not sure you can use the map when running locally. If you set up against your own Firebase instance it should work.
> 
> That said, you should be able to add a point or polygon - just make sure you've selected "Free form" data collection for at least one job when setting up your survey on the web interface.

Ok, let me try doing this. When I loaded the map I was just able to see a blue screen.

**Comment by angrezichatterbox** [comment]
Sorry for the delay in solving this issue. I would get back to it and solve try solving it again.

**Comment by gino-m** [comment]
Just checked, the focus does not automatically get placed on the text box.
